### ISSUE TITLE
Cannot add comments to pull requests by using keyboard only

### ISSUE THREAD

**Issue by marlon-sousa** [microsoft/vscode-pull-request-github#4678]
- Extension version: 0.60.0
- VSCode Version: 1.76.2
- OS: Windows 11 (where vscode is installed), Arch Linux (where vscode remote is being executed)

Hello,

First of all, sorry if this is not considered a bug. I am reporting that because, as I am not able to perform a task my peers are, I consider this a bug.

First of all, I am a visually impaired user, using Visual Studio code on Windows with the NVDA screen reader to access a remote computer running Arch Linux.

I have been using heavily this extension to enhance my productivity when performing code review. I can perform everything I need but adding comments to lines in the pull request.

Steps to Reproduce:

1. Find a pr
2. Press enter on the node or right arrow to expand.
3. Go past the pr toolbar.
4. Navigate pr files.
5. Press enter on one of the files.
6. From this point, I either navigate on the editor where the file is fully shown or press shift + f7 to navigate on the diffs.
7. In both editors I cannot find a way to insert comments on lines I need to. My siighted peers report that by clicking on the plus or less sign in front of the line they access this functionality, but I couldn't, even routing cursors, activating objects, using mouse keys, find anything which would allow me to access the comments editor.
8. I cannot comment on the accessibility of the comments editor because I haven't been able to reach it.

Thanks,
Marlon

**Comment by alexr00** [comment]
@marlon-sousa We have a command "Comments: Add Comment to Current Selection", which you can run from the command palette and will open the comment widget where your cursor is.

If you use VS Code insiders, the command is broken there. I am making a fix now and it should be available next week.
